custom-images/generate_custom_image.py

```diff
@@ -382,13 +388,27 @@ def run():
       """(Optional) The size in GB of the disk attached to the VM instance
       that builds the custom image. If not specified, the default value of
       15 GB will be used.""")
+  parser.add_argument(
```

Comment:
Can you make this flag mutually exclusive with image version?

https://docs.python.org/3/library/argparse.html#mutual-exclusion

Reply:
done